### PR TITLE
Solved Issues in README.md

### PR DESCRIPTION
**Pull request by Mahirgupta**
Fix issues #150 

Changes: The command for starting the postgresql service is add, some typos are corrected.

### CODE REVIEW

**Comment by Mahirgupta** [comment]
Can anyone review this pr ?

File: README.md [hunk 1]
```diff
@@ -5,47 +5,52 @@ This is the official website of the Programming Club at UIET, Panjab University.
 [![Build Status](https://travis-ci.org/pclubuiet/website.svg?branch=master)](https://travis-ci.org/pclubuiet/website)
 
 ## Deploy
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/pclubuiet/website/tree/master)
```
Comment (sagar-kalra) [inline README.md]: Rebase your branch.

Comment (divyam3897) [reply README.md]: why this change? just remove it.

Comment (Mahirgupta) [reply README.md]: which one ?

**Pull request merged**